mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx

```diff
@@ -89,7 +91,7 @@ const ProfilePreview: React.FC = () => {
                             category={TextCategory.BODY1}
                             style={globalStyles.pl10}
                         >
-                            {experienceYears} year of experience
+                            {experienceYearsToDisplay} year of experience
```

Comment:
```suggestion
                            {+experienceYears} year of experience
```

Reply:
experienceYears: number | null, so I have to add {experienceYears && +experienceYears} year of experience